libs/client/src/auth.ts

```diff
@@ -0,0 +1,26 @@
+import { getRestApiUrl } from './config';
+import { dispatchRequest } from './request';
+import { ensureAppIdFormat } from './utils';
+
+export const TOKEN_EXPIRATION_SECONDS = 120;
+
+/**
+ * Get a token to connect to the realtime endpoint.
+ */
+export async function getTemporaryAuthToken(app: string): Promise<string> {
+  const [, appAlias] = ensureAppIdFormat(app).split('/');
+  const token: string | object = await dispatchRequest<any, string>(
```

Comment:
Why is github complaining about `any` here?

Reply:
It's eslint. it does warn you because it prefers you to be strict about typing. But in some cases, like this one, I want it to be any by default.

GitHub is able to render eslint messages in their PRs :)